src/components/DpDataTable/DpTableRow.vue

```diff
@@ -33,7 +33,7 @@
     <td
       v-for="(field, idx) in fields"
       :key="`${field}:${idx}`"
-      :class="{ 'c-data-table__resizable': isTruncatable }"
+      :class="{ 'c-data-table__resizable': isTruncatable } + ' overflow-hidden min-w-[50px]'"
```

Comment:
In the vue 2 PR, there is no `min-w-[50px]` here. Why do we need it here, but not there? Or is it missing there?

Reply:
ah indeed we do not need the `min-w-[50px]` class here, this was added here accidentially. Thanks for the catch, removed the class here https://github.com/demos-europe/demosplan-ui/pull/1093/commits/180af1425462aff2c7c7ea28caddddb9ae6dfd2f